Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/gateway/executorservice/PDClient.java

```diff
@@ -128,17 +126,15 @@ public Response callWebService(Target target, Request request) throws Exception
             // check the policy for the outgoing request to the target community
             boolean bIsPolicyOk = checkPolicy(newRequest, assertion);
             if (bIsPolicyOk) {
-                // Audit the patientDiscovery Request Message sent on the Nhin Interface
-                PatientDiscoveryAuditor auditLog = new PatientDiscoveryAuditLogger();
-                auditLog.auditEntity201305(newRequest, assertion, NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION);
+                //Removed the old PatientDiscoveryAuditLogger. 
```

Comment:
Since you're already going to need another commit, can you kill the trailing space at the end of this line?